workspace/run_sft_tarred.sh

```diff
@@ -0,0 +1,46 @@
+NEMO_DIR=/home/heh/codes/nemo-zhehuai
```

Comment:
could you test locally to make sure both tar and untar cases work

Reply:
fixed sample_alpha and tested on both tar and untar datasets